examples/src-jvm/SelfHosted2048Bot.scala

```diff
@@ -1,10 +1,8 @@
 import akka.http.scaladsl.model.Uri
 import akka.http.scaladsl.model.Uri.{Path, Query}
-import akka.http.scaladsl.server.Directives._
```

Comment:
Sorry I was not clear in my latest answer. We can keep the import here, we only need to replace `akka.http.scaladsl.server.Directives._` by `akka.http.scaladsl.server.RouteConcatenation._`

Reply:
I tried that in this file, and it didn't compile. Worked fine in the other file though.

Reply:
Oh yes, some others imports are missing.
I would really prefer avoid having those import inside the class itself, you should be able to remove them and replace them by 
```
import akka.http.scaladsl.server.RouteConcatenation._
import akka.http.scaladsl.server.Directives.{pathPrefix, getFromResourceDirectory}
import akka.http.scaladsl.server.Route
```
At the top of the file

Once done I think this pull request will be ready to be merged into master (I don't think I'll create a 2.13 branch because this lib must be updated anyway and will probably have a lot of breaking changes in the next few weeks).

Reply:
Alright, I pushed the changes. 

Sorry, IntelliJ doesn't work too well with this project and I'm not quite used to VSCode with Metals yet, so I'm a bit awkward with tooling etc.